flink-table/flink-sql-gateway/src/test/java/org/apache/flink/table/gateway/service/MaterializedTableStatementITCase.java

```diff
@@ -197,6 +220,29 @@ void testCreateMaterializedTableInContinuousMode() throws Exception {
                 .isEqualTo(CatalogMaterializedTable.RefreshStatus.ACTIVATED);
         assertThat(actualMaterializedTable.getRefreshHandlerDescription()).isNotEmpty();
         assertThat(actualMaterializedTable.getSerializedRefreshHandler()).isNotEmpty();
+
+        ContinuousRefreshHandler continuousRefreshHandler =
+                ContinuousRefreshHandlerSerializer.INSTANCE.deserialize(
+                        actualMaterializedTable.getSerializedRefreshHandler(),
+                        getClass().getClassLoader());
+        // check the background job is running
+        String describeJobDDL =
+                String.format("DESCRIBE JOB '%s'", continuousRefreshHandler.getJobId());
+        OperationHandle describeJobHandle =
+                service.executeStatement(sessionHandle, describeJobDDL, -1, new Configuration());
+        awaitOperationTermination(service, sessionHandle, describeJobHandle);
+        List<RowData> jobResults = fetchAllResults(sessionHandle, describeJobHandle);
+        assertThat(jobResults.get(0).getString(2).toString()).isEqualTo("RUNNING");
+
+        // get checkpoint config from the materialized table
+        CheckpointConfigInfo checkpointConfigInfo =
+                getCheckpointConfigInfo(clusterClient, continuousRefreshHandler.getJobId());
```

Comment:
Can we get the JSON string directly? It looks a bit strange to convert the CheckpointConfigInfo to a json string, and then read the interval from json tree.

Reply:
Currently, I have not found a good way to directly obtain the response of type String.

A better way might be to add a get method to CheckpointConfigInfo class.

Reply:
ok